config/cops.yml

```diff
@@ -1 +1,24 @@
-Metrics/ClassLength: 400_000
+Metrics/AbcSize:
+  violation_cost: 100_000
```

Comment:
This is my fault for not mentioning, but these are `_points` not `_cost` these days :)